backend/pkg/database/odatasql/query.go

```diff
@@ -23,8 +23,14 @@ import (
 
 	"github.com/CiscoM31/godata"
 	log "github.com/sirupsen/logrus"
+
+	"github.com/openclarity/vmclarity/backend/pkg/database/odatasql/jsonsql"
 )
 
+var SQLVariant jsonsql.Variant = jsonsql.SQLite{}
```

Comment:
It was just the default so that the odatasql library worked without the need to override if you were using SQLite, but I've updated the code so that the variant is passed in at runtime by whoever is using the library. Its much better to do that than have global variables anyway.